app/Commands/StartCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Commands;
+
+use App\InitializesCommands;
+use App\Services;
+use App\Shell\Docker;
+use Illuminate\Support\Str;
+use LaravelZero\Framework\Commands\Command;
+use PhpSchool\CliMenu\CliMenu;
+
+class StartCommand extends Command
+{
+
+    use InitializesCommands;
+
+    protected $signature = 'start {container?}';
+    protected $description = 'Start a stopped container.';
+
+    public function handle(): void
+    {
+        $this->initializeCommand();
+
+        $container = $this->argument('container');
+
+        if ($container) {
+            $this->start($container);
+
+            return;
+        }
+
+        $this->menu('Containers to start')
+            ->addItems($this->startableContainers())
+            ->open();
+    }
+
+    public function startableContainers(): array
+    {
+        return collect(app(Docker::class)->takeoutContainers())->skip(1)->reduce(function ($containers, $container) {
+            if(!Str::contains($container[2], 'Up')) {
```

Comment:
Can we move this `Str::contains` into a `reject` before we get into the reduce?

Reply:
Also... could this become a `mapWithKeys` instead of a `reduce`? What we're basically doing here is rejecting a bunch of containers then modifying the rest, right?

Reply:
Is there any reason to use mapWithKeys instead of just map?

Reply:
I think I thought you were ending up with a key and a value; if `map` makes more sense then I probably just read it wrong!